### ISSUE TITLE
Server crashes while loading the colonie

### ISSUE THREAD

**Issue by Inomutso** [ldtteam/minecolonies#2928]
### Minecolonies version

Version: minecolonies-1.12.2-0.9.141-ALPHA-universal.jar

### Expected behavior

- start server without crash

### Actual behaviour

- server crashes while loading the colonie

< crash-2018-10-01_23.47.47-server.txt (just one of the crash logs)
[crash-2018-10-01_23.47.47-server.txt](https://github.com/ldtteam/minecolonies/files/2439715/crash-2018-10-01_23.47.47-server.txt)

### Steps to reproduce the problem

1. happens everytime the server starts

Hi Team

Yesterday I changed a value for the Request System, using the Mod-Option Button in the MC Launcher, while playing

Should the request system creatively resolve (if possible) when the player is required to resolve a request.
B:creativeResolve=true
		
from false to true
later on I had to restart our hosted Server and thince then, the Server is crashing every time I try to start

java.lang.IllegalArgumentException: Already a colony registered to id=1, colony=Inomutso's Colony, not changing to colony=Inomutso's Colony

I tried several things to get the server up again, but no chance, everytime the same issue

Do you think there is any chance to get back to our colonie, without losing all the changes and buildings, thince our last Server Backup ?
our colonie is one year old (60 citizens) we have gone thru all version and are still happy with your great Mod

that happend when the Server was running minecolonies-1.12.2-0.9.139-ALPHA-universal.jar
meanwhile I switched to minecolonies-1.12.2-0.9.141-ALPHA-universal.jar, but that has no change on this issue

any idea on this ? any hints ?  thx

**Comment by Nightenom** [comment]
if you can send us your colony backup (make a zip of recent .dat files in minecolonies folder, it's in the world folder I guess) we can probably solve it, it's trying to register the same colony twice for an unknown reason

**Comment by Inomutso** [comment]
Hi Team

As requested, this is our complete colony folder that is stored in the /world folder on the server
Also i wanted to show you a logfile that we have when the server is up, where you can find some entries like

[21:13:36] [Server thread/WARN] [net.minecraft.world.WorldServer]: Keeping entity minecraft:chicken that already exists with UUID 3dc3b5d9-53b4-4b82-b4b7-55a43d843bd9

this one we have already for a long time (month), maybe it´s possible to find that chicken and kill it

these ones came up when updating to a release like
minecolonies-1.12.2-0.9.1-RELEASE-universal.jar
or
minecolonies-1.12.2-0.9.7-BETA-universal.jar
can´t remember ist exactly 

[21:09:32] [Server thread/INFO] [minecolonies]: Teleported stuck citizen Scarlett R. Cosworth from colony: 1 to target location

[21:09:39] [Server thread/INFO] [minecolonies]: Teleported stuck citizen Philip R. Gonson from colony: 1 to target location

these two and others in the logfile are Guards, patroling around, but i cannot see them getting stuck
apologies for my english, never learned it properly, we are in Germany
we hope (my kid´s and me) that you can help us with this

thanks for your quick reply Nightenom and best regards Inomutso (Frank)

[2018-09-25-3.log.gz](https://github.com/ldtteam/minecolonies/files/2441388/2018-09-25-3.log.gz)
[20181003_minecolonies_Inomutso.zip](https://github.com/ldtteam/minecolonies/files/2441389/20181003_minecolonies_Inomutso.zip)

**Comment by someaddons** [comment]
I've had that issue aswell after updating versions, just deleted my colony1.dat since stuff is in capabilities aswell

**Comment by Nightenom** [comment]
I will take a look into files you send somewhen from thursday to sunday. The first error with uuid you can just skip, the errors about teleporting stuck citizen - you should look into your colony if they can path freely (doors, walls, etc.), you can use waypoints via the buildtool too, whatever you don't need to worry about these errors too

**Comment by Inomutso** [comment]
Hi someaddons, Nightenom

I tried that, deleted my colony1.dat and the server starts up again

Before I tried to change colonies.dat and colony1.dat to older versions,
but that did not help. So we are very happy with your hint !

just wondering, what this file was doing, because the colony did not
generate a new one and is still living without one.  

I have rebooted the server 3 times, but everything keeps beeing ok,
we did not loose anything ingame.

thanks again from the team & kind regards
